packages/ffe-cards-react/src/GroupCard/GroupCard.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { GroupCard } from './GroupCard';
+import { render, screen } from '@testing-library/react';
+import { GroupCardElement } from './GroupCardElement';
+
+describe('GroupCard', () => {
+    it('should render correct class', () => {
+        render(
+            <GroupCard>
+                <div>Test</div>
+            </GroupCard>,
+        );
+        const card = screen.getByRole('group');
+        expect(card.classList.contains('ffe-group-card')).toBeTruthy();
+    });
+
+    it('should render GroupCardElement with correct class when provided as child', () => {
+        render(
+            <GroupCard>
+                <GroupCardElement>Hello world</GroupCardElement>
+            </GroupCard>,
+        );
+        const cardGroup = screen.getByRole('group');
+        const cardGroupElement = cardGroup.querySelector(
+            '.ffe-group-card__element',
+        );
+        expect(cardGroupElement?.textContent).toBe('Hello world');
+    });
+
+    it('should render a custom class', () => {
```

Comment:
Den her hade jag nok bakat in i førsta testen.. fint og teste en ting åt gagngen ,en da får man testet att custom classen ikke skriver over `ffe-group-card` hvis den er brukt.